pkg/daemon/ceph/client/pool_test.go

```diff
@@ -150,19 +150,29 @@ func TestSetPoolApplication(t *testing.T) {
 }
 
 func TestCreateReplicaPoolWithFailureDomain(t *testing.T) {
-	testCreateReplicaPool(t, "osd", "mycrushroot", "", "")
+	testCreateReplicaPool(t, "osd", "mycrushroot", "", "", DefaultPGCount, DefaultAutoscaleMode)
+}
+
+func TestCreateReplicaPoolWithPgs(t *testing.T) {
+	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "", "64", DefaultAutoscaleMode)
+}
+
+func TestCreatePrelicaPoolWithAutoscaleMode(t *testing.T) {
+	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "", DefaultPGCount, "on")
+	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "", DefaultPGCount, "off")
+	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "", DefaultPGCount, "warn")
 }
 
 func TestCreateReplicaPoolWithDeviceClass(t *testing.T) {
-	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "")
+	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "", DefaultPGCount, DefaultAutoscaleMode)
 }
 
 func TestCreateReplicaPoolWithCompression(t *testing.T) {
-	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "passive")
-	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "force")
+	testCreateReplicaPool(t, "osd", "mycrushroot", "hdd", "passive", DefaultPGCount, DefaultAutoscaleMode)
```

Comment:
How about adding a test that does not use the default pg count?

Reply:
line 157 does not use the default pg count.